api/v1alpha1/metrics.go

```diff
@@ -5,8 +5,9 @@ import (
 )
 
 type Metrics struct {
-	Total   uint64            `json:"total,omitempty" protobuf:"varint,1,opt,name=total"`
-	Errors  uint64            `json:"errors,omitempty" protobuf:"varint,2,opt,name=errors"`
-	Rate    resource.Quantity `json:"rate,omitempty" protobuf:"bytes,3,opt,name=rate"` // current rate of messages per second
-	Retries uint64            `json:"retries,omitempty" protobuf:"bytes,4,opt,name=retries"`
+	Total      uint64            `json:"total,omitempty" protobuf:"varint,1,opt,name=total"`
+	Errors     uint64            `json:"errors,omitempty" protobuf:"varint,2,opt,name=errors"`
+	Rate       resource.Quantity `json:"rate,omitempty" protobuf:"bytes,3,opt,name=rate"` // current rate of messages per second
+	Retries    uint64            `json:"retries,omitempty" protobuf:"bytes,4,opt,name=retries"`
+	TotalBytes uint64            `json:"totalBytes,omitempty" protobuf:"varint,5,opt,name=totalBytes"`
```

Comment:
Bytes will reach uint64 max soon. @alexec  Can we have TotalKiloBytes or TotalMBytes?

Reply:
try `resource.Quantity`

Reply:
I will keep uint64. resource.Quantity has int64

Reply:
is `uint64` big enough do you think?  18446744073709551615 in max uint64

![image](https://user-images.githubusercontent.com/1142830/129937344-4710df7d-818d-4bc5-b34e-d8aaee9f8e82.png)
![image](https://user-images.githubusercontent.com/1142830/129937420-3e0ecf2b-903b-41cd-882e-624168f9d5d5.png)

@whynowy do you want to add your thoughts?